### ISSUE TITLE
multiqc error using dev branch

### ISSUE THREAD

**Issue by rrdavis77** [nf-core/viralrecon#345]
### Description of the bug

Hi,

I had issues running a few samples through viralrecon 2.5 with ivar_variants_to_vcf.py so I decided to run it on the dev branch and it made it past those errors but received errors running multiqc

### Command used and terminal output

```console
NXF_VER=21.10.3 nextflow run nf-core/viralrecon --input samplesheet_all_samples_10.csv --platform illumina --genome NC_045512.2 --protocol amplicon --primer_bed /share/sequencing_effort/viralrecon/primer_files/NC_045512.2.v2.primers.bed --primer_fasta /share/sequencing_effort/viralrecon/primer_files/NC_045512.2.v2.primers.fna --skip_assembly -profile singularity --primer_left_suffix F --primer_right_suffix R --kraken2_db /share/sequencing_effort/viralrecon/krakendb/kraken2_human.tar.gz --ivar_trim_offset 5 -c /share/sequencing_effort/viralrecon/viralrecon_dev.config --outdir /share/Viralrecon_results/test_dev/ -r dev -resume --skip_asciigenome --skip_snpeff --skip_kraken2 --skip_pangolin --skip_nextclade

Error reported:

Command exit status:
  1

Command output:
  (empty)

Command error:
  INFO:    Convert SIF file to sandbox...
                  | v1.13.dev0
  
  |           multiqc | Only using modules: custom_content, fastqc, fastp, kraken, bowtie2, samtools, mosdepth, bcftools, snpeff, quast, pangolin, cutadapt
  |           multiqc | Search path : /share/Viralrecon_results/test_dev/work/dd/88a60fee0466614174d1a6ec991086
  |         searching | ���������������������������������������� 100% 102/102  
  |    custom_content | software_versions: Found 1 sample (html)
  |    custom_content | nf-core-viralrecon-summary: Found 1 sample (html)
  |    custom_content | amplicon_heatmap: Found 10 samples (heatmap)
  �������������� Oops! The 'custom_content' MultiQC module broke... ��������������
  � Please copy this log and report it at                                        �
  � https://github.com/ewels/MultiQC/issues                                      �
  � Please attach a file that triggers the error. The last file found was:       �
  � ./variants/4056145117.variant_counts_mqc.tsv                                 �
  �                                                                              �
  � Traceback (most recent call last):                                           �
  �   File "/usr/local/lib/python3.10/site-packages/multiqc/multiqc.py", line 65 �
  �     output = mod()                                                           �
  �   File "/usr/local/lib/python3.10/site-packages/multiqc/modules/custom_conte �
  �     parsed_modules[mod_id].add_cc_section(c_id, mod)                         �
  �   File "/usr/local/lib/python3.10/site-packages/multiqc/modules/custom_conte �
  �     mod["data"] = {k: v for k, v in sorted(mod["data"].items())}             �
  � TypeError: '<' not supported between instances of 'str' and 'float'          �
  �                                                                              �
  ��������������������������������������������������������������������������������
  |            fastqc | Found 20 reports
  |             fastp | Found 10 reports
  |           bowtie2 | Found 10 reports
  |          samtools | Found 10 flagstat reports
  |          samtools | Found 10 flagstat reports
  |          mosdepth | Found 10 reports
  |          bcftools | Found 10 stats reports
  |             quast | Found 10 reports
  |           multiqc | Compressing plot data
  |           multiqc | Report      : multiqc_report.html
  |           multiqc | Data        : multiqc_data
  |           multiqc | MultiQC complete
  INFO:    Cleaning up image...

Work dir:
  /share/Viralrecon_results/test_v2.5/work/dd/88a60fee0466614174d1a6ec991086

Tip: you can replicate the issue by changing to the process work dir and entering the command `bash .command.run`




Is the issue with the header of the file attached?
```

### Relevant files

[4056145117.variant_counts_mqc.tsv.zip](https://github.com/nf-core/viralrecon/files/10147491/4056145117.variant_counts_mqc.tsv.zip)

### System information

Nextflow version 21.10.3
Hardware HPC
Executor local
Container engine: Singularity
OS Ubuntu Linux
Version of nf-core/viralrecon dev

**Comment by drpatelh** [comment]
Hi @rrdavis77 ! Are you able to try the latest `dev` code please? I have just bumped the MultiQC version to 1.14 which may or may not solve the issue. If it doesn't, be great if you can share a minimal reproducible example so I can try to fix.

```
nextflow pull nf-core/viralrecon -r dev
nextflow run nf-core/viralrecon -r dev <YOUR_PARAMETERS>
```

**Comment by rrdavis77** [comment]
Surething!  I will give it a spin tomorrow and report back here

**Comment by drpatelh** [comment]
Thanks! Last remaining issue to figure out before the release 🤞🏽

**Comment by rrdavis77** [comment]
bummer, got the exact same error while testing the latest dev on these 10 samples.  I tried a combination of 1,2 or 3 samples and I can't get it to fail.  I will try a few more combos tomorrow morning.  When it errors, it complains about the same sample above.  If I run the "problematic" sample by itself, it passes.
If I can get this down to 5 samples or so, how can I share the raw data with you?  do you use box?

**Comment by rrdavis77** [comment]
ok, headed to bed but my latest failed with just two samples and it seems to fail when I include two samples named in different manner and that might be causing the issue.  I have samples that are named with numerical digits and samples that start with FS followed by digits such as:
```
4056145141
FS25388291
```
that seems to be causing the issue with multiqc

**Comment by rrdavis77** [comment]
@drpatelh I can confirm that the name of the samples causes this issue.  I took two samples that only had digits in the sample name and the pipeline completed.  I took the same samplesheet and modified one of the sample names to start with FS and the error above appeared.  You can probably duplicate this bug by changing your filenames to have one samples with only digits and the other with letters.  Do I need to submit an issue with multiqc?

**Comment by drpatelh** [comment]
Thank you very much for reproducing this down to the name @rrdavis77 ! Yes, please. Be great if you can submit a bug to MultiQC repo. Do you know which file out of all of the ones used by MultiQC triggers the error? Phil might ask for that to be able to fix.

**Comment by rrdavis77** [comment]
No problem!  I will create an issue for Phil to look at.  It seems to hang on these files : *.variant_counts_mqc.tsv

**Comment by drpatelh** [comment]
Thanks! In the meantime, you know what to can't do! 😅 I'll look out for a fix in future releases and keep this open.

**Comment by rrdavis77** [comment]
Seeing that there is work on 2.7 hapening, I wanted to doublecheck that the above issue was resolved.  Using multiqc 1.19 instead of 1.14 in 2.6, the issue above was resolved.

**Comment by svarona** [comment]
This was fixed in PR #413 when updating from multiqc 1.14 to 1.19.
